src/utils/api.js

```diff
@@ -0,0 +1,11 @@
+export const api = async (url, method) => {
```

Comment:
Do you think this implementation will work for all http verbs?

Reply:
No this implementation doesn't with all http werbs. I have only implemented this api util only for GET requests, but it is possible to add the functionality for all http verbs.